config/ci_configs/amip_coarse_ft64_hourly_checkpoints_restart.yml

```diff
@@ -1,6 +1,6 @@
 apply_limiter: false
 co2: "maunaloa"
-checkpoint_dt: "1hours"
+checkpoint_dt: "400secs"
 dt_save_restart: "10days"
```

Comment:
`dt_save_restart` isn't used in ClimaCoupler, and it doesn't appear in any other configs. Could you remove it from here please?